packages/database/src/migrations/20210625012001-FixCustomLegacyReports-modifies-data.js

```diff
@@ -0,0 +1,230 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+const LEGACY_REPORTS = [
+  {
+    code: 'Disaster_Response_Comparisons',
+    oldDataBuilderConfig: {
+      dataElementPairs: [['DP_NEW001', 'DP9']],
+    },
+    newDataBuilderConfig: {
+      dataElementPairs: [['DP_NEW001', 'DP9']],
+      leftColumnHeader: 'Normal',
+      rightColumnHeader: 'Current',
+      dataPairNames: ['Inpatient beds'],
+    },
```

Comment:
A bit of context, there are actually a few legacy data builders that reference `viewJson`. They have access to it because the previous `view` endpoint queries it. With the new `report` endpoint which should only reference either the modern report or the legacy report tables, it no longer queries for `viewJson` (now called `config` in dashboard_item). 

Most of the `viewJson` references in these legacy data builders are just hardcoded strings which is required to calculate the data. So what I did here is just move whatever they were referencing from `viewJson` before to `data_builder_config`. It is not ideal, but I feel that since these data builders are legacy, I really don't think it's worth to spend time to refactor them to fit the new endpoint structure.